controllers/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -99,6 +98,11 @@ func (r *SnapshotEnvironmentBindingReconciler) Reconcile(ctx context.Context, re
 
 	log.Info(fmt.Sprintf("Starting reconcile loop for %v %v", appSnapshotEnvBinding.Name, req.NamespacedName))
 
+	ghClient, err := r.GitHubTokenClient.GetNewGitHubClient()
+	if err != nil {
+		log.Error(err, "Unable to create Go-GitHub client due to error")
```

Comment:
ghclient is nil at this point. We should return and stop processing

Reply:
Fixed